script.js

```diff
@@ -24,8 +29,19 @@ function checkEnd(penguinPosition) {
     return penguinPosition > 1000
 }
 
-function endGameMessage() {
-    alert('Far out brah! That was totally radical, you found some sweet lines down those faces! Catch you on the flip')
+function saveResult(playerName, playerScore) {
+    let playerSaveObject = {name: playerName, score: playerScore}
+    localStorage.setItem('playerKey', JSON.stringify(playerSaveObject))
+    alert('Score Saved!')
+}
+
+function getName(playerName) {
+    if (playerName.length === 3) {
+        saveResult(playerName, timer)
+    }
+    else {
+        alert('Please use a three character name!')
+    }
 }
```

Comment:
(as discussed) i think it would make more sense if:
- `getName` is called `validateName` and it returns true or false
- instead of `validateName` calling `saveResult`, `saveResult` calls `validateName`
- the alerts are in `saveResult`